### PR TITLE
[env/burn-staging] add photobooth link

### PR DESCRIPTION
**Pull request by malaikaaiyar**
Screen Shot 2021-09-02 at 10 04 35 PM

fixes https://github.com/sparkletown/internal-sparkle-issues/issues/1142

### CODE REVIEW

File: src/components/molecules/NavBar/NavBar.tsx [hunk 1]
```diff
@@ -313,6 +313,17 @@ export const NavBar: React.FC<NavBarPropsType> = ({
                 <div>{navbarTitle}</div>
               )}
 
+              <div
+                className="nav-schedule nav-photobooth"
+                onClick={() =>
+                  openUrlInNewTab(
+                    "https://burn.sparklever.se/in/outsnappedphotoboothcamp"
```
Comment (goran-peoski-work) [inline src/components/molecules/NavBar/NavBar.tsx:320]: can extract constant for the URL

File: src/components/molecules/NavBar/NavBar.tsx [hunk 2]
```diff
@@ -313,6 +313,17 @@ export const NavBar: React.FC<NavBarPropsType> = ({
                 <div>{navbarTitle}</div>
               )}
 
+              <div
+                className="nav-schedule nav-photobooth"
```
Comment (goran-peoski-work) [inline src/components/molecules/NavBar/NavBar.tsx:317]: Easy win, some times no need of styles, but here really easy to update the SCSS also

```suggestion
                className="NavBar__photoboot-button nav-schedule nav-photobooth"
```

File: src/components/molecules/NavBar/NavBar.tsx [hunk 3]
```diff
@@ -313,6 +313,17 @@ export const NavBar: React.FC<NavBarPropsType> = ({
...
+                onClick={() =>
+                  openUrlInNewTab(
+                    "https://burn.sparklever.se/in/outsnappedphotoboothcamp"
+                  )
+                }
+              >
+                <p>Photobooth</p>
```
Comment (goran-peoski-work) [inline src/components/molecules/NavBar/NavBar.tsx:324]: This way, no need to target `p` if by any chance it needs to be replaced with `div` that has no styles in `global.scss` set

```suggestion
                <p className="NavBar__photoboot-title">Photobooth</p>
```

File: src/components/molecules/NavBar/NavBar.scss [hunk 4]
```diff
@@ -242,6 +245,24 @@ $simplified-view-button--padding: 0 10px;
...
+    background-color: $lighter-intermediate-grey;
+  }
+
+  p {
+    text-decoration: none;
+    margin: $photobooth-button-text-margin;
+    font-size: $font-size--sm;
+  }
+}
+
```
Comment (goran-peoski-work) [inline src/components/molecules/NavBar/NavBar.scss:265]: Selectors can be more specific using BEM

```scss
.NavBar__photoboot {
  // ...
  &-button {
     // ...
  }
  &-button:hover  {
    // ...
  }
  &-title {
    // ...
  }
}
```

**Review by goran-peoski-work — CHANGES_REQUESTED** [review]
Minor class/scss selector adjustments

File: src/components/molecules/NavBar/NavBar.tsx (hunk 1 above)
Comment (whoaitsaimz) [reply src/components/molecules/NavBar/NavBar.tsx:320]: Why new tab? I don't think that's right.

File: src/components/molecules/NavBar/NavBar.tsx (hunk 2 above)
Comment (whoaitsaimz) [reply src/components/molecules/NavBar/NavBar.tsx:317]: photoboot-button = photoboot**h**-button ;-)

File: src/components/molecules/NavBar/NavBar.scss (hunk 4 above)
Comment (whoaitsaimz) [reply src/components/molecules/NavBar/NavBar.scss:265]: photoboot = photobooth (missing h)

Comment (malaikaaiyar) [reply src/components/molecules/NavBar/NavBar.scss]: fixed in cab515897d168494475f82a5a0970d9270d88094

File: src/components/molecules/NavBar/NavBar.tsx (hunk 3 above)
Comment (malaikaaiyar) [reply src/components/molecules/NavBar/NavBar.tsx]: fixed in cab515897d168494475f82a5a0970d9270d88094

File: src/components/molecules/NavBar/NavBar.tsx (hunk 2 above)
Comment (malaikaaiyar) [reply src/components/molecules/NavBar/NavBar.tsx]: fixed in cab515897d168494475f82a5a0970d9270d88094

File: src/components/molecules/NavBar/NavBar.tsx (hunk 1 above)
Comment (malaikaaiyar) [reply src/components/molecules/NavBar/NavBar.tsx]: fixed in cab515897d168494475f82a5a0970d9270d88094

**Review by goran-peoski-work — APPROVED** [review]
LGTM

**Pull request merged**